deploy/helm-chart/crds/gateway.nginx.org_nginxproxies.yaml

```diff
@@ -0,0 +1,162 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+  name: nginxproxies.gateway.nginx.org
+spec:
+  group: gateway.nginx.org
+  names:
+    kind: NginxProxy
+    listKind: NginxProxyList
+    plural: nginxproxies
+    singular: nginxproxy
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: NginxProxy represents the dynamic configuration for an NGINX
+          Gateway Fabric data plane.
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: NginxProxySpec defines the desired state of the NginxProxy.
+            properties:
+              http:
+                description: Http defines the Http configuration.
```

Comment:
If we make the change above that @pleshakov suggested about Http -> HTTP, should change here too.